Back-End/controllers/registration_controllers/login_controllers.js

```diff
@@ -1,5 +1,38 @@
-const db = require('../../db');
+const connection = require('../../db');
+require("dotenv").config();
+const bcrypt = require('bcrypt');
+const jwt = require('jsonwebtoken');
+
+
+const login = async (req,res)=>{
+    let email=req.body.email;
+    const query = await `SELECT * FROM users WHERE email ='${email}'`
+
+    connection.query(query,async(err,result)=>{
+        if(err) throw err;
+        
+        //check if there is user with the request data
+        if(result.length===1) {
```

Comment:
you could just change it to `if (result.length)`  if the length is 0 then it is a falsy value anything other than 0 means that there is a result and any number expect 0 is a truthy number